-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

- From b88b006591774dede84b1692b7b3906d712ed20a Mon Sep 17 00:00:00 2001
From: Sebastian Bach <sebtomba@gmail.com>
Date: Wed, 24 May 2023 12:42:44 +0200
Subject: [PATCH] [PM-6226] Rename rename module mn_api to midnight_api

- ---
 node/src/midnightrpc.rs                    | 2 +-
 node/src/rpc.rs                            | 2 +-
 runtime/src/lib.rs                         | 4 ++--
 runtime/src/{mn_api.rs => midnight_api.rs} | 0
 4 files changed, 4 insertions(+), 4 deletions(-)
 rename runtime/src/{mn_api.rs => midnight_api.rs} (100%)

diff --git a/node/src/midnightrpc.rs b/node/src/midnightrpc.rs
index 36621e20..0a03feac 100644
- --- a/node/src/midnightrpc.rs
+++ b/node/src/midnightrpc.rs
@@ -6,7 +6,7 @@ use std::{
 use jsonrpsee::{core::RpcResult, proc_macros::rpc, types::error::CallError};
 use ledger::structure::Transaction;
 use midnight_base_crypto::proofs::Proof;
- -use node_template_runtime::mn_api::MidnightRuntimeApi;
+use node_template_runtime::midnight_api::MidnightRuntimeApi;
 use sc_client_api::BlockBackend;
 use sp_api::ProvideRuntimeApi;
 use sp_blockchain::HeaderBackend;
diff --git a/node/src/rpc.rs b/node/src/rpc.rs
index 92105a0c..59553364 100644
- --- a/node/src/rpc.rs
+++ b/node/src/rpc.rs
@@ -15,7 +15,7 @@ use sp_api::ProvideRuntimeApi;
 use sp_block_builder::BlockBuilder;
 use sp_blockchain::{Error as BlockChainError, HeaderBackend, HeaderMetadata};
 
- -use node_template_runtime::mn_api::MidnightRuntimeApi;
+use node_template_runtime::midnight_api::MidnightRuntimeApi;
 pub use sc_rpc_api::DenyUnsafe;
 
 /// Full client dependencies.
diff --git a/runtime/src/lib.rs b/runtime/src/lib.rs
index 0d483c16..b69c8d95 100644
- --- a/runtime/src/lib.rs
+++ b/runtime/src/lib.rs
@@ -2,7 +2,7 @@
 // `construct_runtime!` does a lot of recursion and requires us to increase the limit to 256.
 #![recursion_limit = "256"]
 
- -pub mod mn_api;
+pub mod midnight_api;
 
 use codec::{Decode, Encode};
 use pallet_grandpa::{
@@ -319,7 +319,7 @@ impl_runtime_apis! {
 		}
 	}
 
- -	impl mn_api::MidnightRuntimeApi<Block> for Runtime {
+	impl midnight_api::MidnightRuntimeApi<Block> for Runtime {
 		fn get_contract_state(contract_addres: [u8;32]) -> Vec<u8>{
 			MidnightModule::get_contract_state(contract_addres)
 		}
diff --git a/runtime/src/mn_api.rs b/runtime/src/midnight_api.rs
similarity index 100%
rename from runtime/src/mn_api.rs
rename to runtime/src/midnight_api.rs
- -- 
2.46.0

-----BEGIN PGP SIGNATURE-----

iQIzBAEBCAAdFiEE9Oz16eF+Nv5C7TpSdEmbFI1hGmwFAmhtArMACgkQdEmbFI1h
Gmx2pRAAkERD9wtAHwvtfTdiuU3Dj15xw/e87zsB01tSklSLwgaj3o4XMlVrwLzA
hp0Uxmn8z3ZrjtR/xaiGKkEcRT+3rVlSsCWIAsmNawhwRU4mM+WAhclxbNc6VEmx
La7Qm8i1Xcr2uz6pYk1olKAXrKTIwtHcOAkxeYo0O5QcLkTVyYtPdZ5hrK4wdK4h
+y7pxgUQDlNh+zIPHMayckD8+5xSMFuhBgKaVYq1NQWUyNmTy6OgV05F5cgHRNX6
t9yZMTYEDJ/e48Cvrgk7MDbV5VX3EwoyjdtJHaNy5Kn56OloqJzHIgyaYkWgckTp
GiKyCUGkod5cKkUI9PnY9sFIbCilpJDM1bNza+zOS5Gnav3zPrKfolDQBJ63pvpu
0AUvNXoFlKyHVaEyyfn8q+lJTv5O5Ugxa1nRZqwFkh8baQidN6U5rQDi/loiUuXN
JX/NTpZ+syv7GjbegPzwMYuwafTfIcJWbymdfhLJY9oPXjk6iOt521LtmBsuK6n8
Gj/qKteWK5oqUGLwWuoVx6rjfLLLIPD5iXCqjNjXHlQcZlIk+xLKKhZDuKapG5Df
X0F9d+Z5hkqWtiBuylThxRKDmw1pKSUNKW24j1dIYl1saG4U2AIrN9E1bFRt6Chj
cBUGAh1ZScrrTdREELl6NOScoGuSISpTplcIhR4wVTz11xpTd0I=
=m67c
-----END PGP SIGNATURE-----
